netlogo-gui/src/main/app/codetab/CodeTab.scala

```diff
@@ -50,7 +54,20 @@ class CodeTab(val workspace: AbstractWorkspace) extends JPanel
   override def zoomTarget = text
 
   val errorLabel = new EditorAreaErrorLabel(text)
-  val toolBar = getToolBar
+  val toolBar = new ToolBar {
```

Comment:
Why having a `val` that just calls the function? What do you gain by putting it in a function?